scripts/release/deploy_staging_jars.sh

```diff
@@ -38,6 +38,7 @@ fi
 
 declare -a ALL_VERSION_OPTS=(
 "-Dscala-2.11 -Dspark2.4 -Dflink1.13"
+"-Dscala-2.11 -Dspark2.4 -Dflink1.14"
```

Comment:
Do we need to add `-Dspark3` related entry as well?

Reply:
good point. i should add that

Reply:
@yihua a different thought: `spark3` was meant for legacy bundle name but it actually is built with spark 3.2 in this release, which is different from previous `spark3` bundle which is built with spark 3.1. So the compatibility does not exist despite the same name. We should reconsider the bundle jar name changes: it might be more confusing if we publish BWC bundle jar names along with the new name format `sparkX.Y-bundle`: user may be wondering what they suppose to use. Let's discuss when we have a chance to chat.

Reply:
Per discussion, we'll still keep `spark3` bundle name as the legacy one and clearly document what it refers to.